libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2027,6 +2027,32 @@ void TypeChecker::typeCheckABIEncodeFunctions(
 	}
 }
 
+void TypeChecker::typeCheckBytesConcatFunction(
+	FunctionCall const& _functionCall,
+	FunctionType const* _functionType
+)
+{
+	solAssert(!!_functionType, "");
```

Comment:
```suggestion
	solAssert(_functionType, "");
```
Also fine with keeping.

Reply:
Fixed

Reply:
Fixed